### PR TITLE
UILD-282: repeatable subcomponents

### PR DESCRIPTION
**Pull request by SKarolFolio**
[https://folio-org.atlassian.net/browse/UILD-282](https://folio-org.atlassian.net/browse/UILD-282)

### CODE REVIEW

File: src/common/helpers/repeatableFields.helper.ts [hunk 1]
```diff
@@ -28,3 +29,30 @@ export const checkRepeatableGroup = ({
...
+    constraints?.repeatable &&
+    // show "Duplicate" button only for Provision Activity's subcomponents
+    // TODO: change or remove to display "Duplicate" button for other groups
+    findParentEntryByProperty({
+      schema,
+      path,
+      key: 'uriBFLite',
+      value: BFLITE_URIS.PROVISION_ACTIVITY,
+    }) &&
+    // TODO: remove this condition after updating the profile
```
Comment (askhat-abishev) [inline src/common/helpers/repeatableFields.helper.ts:54]: isn't the profile already was updated?

File: src/components/DuplicateGroup/DuplicateGroup.tsx [hunk 2]
```diff
@@ -7,15 +8,19 @@ import './DuplicateGroup.scss';
...
+export const DuplicateGroup: FC<Props> = memo(({ onClick, hasDeleteButton = true, className }) => (
+  <div className={classNames(['duplicate-group', className])}>
     <Button data-testid="id-duplicate-group" type={ButtonType.Icon} onClick={onClick}>
       <Plus16 />
     </Button>
-    <Button data-testid="id-delete-duplicate-group" type={ButtonType.Icon} disabled={IS_DISABLED_FOR_ALPHA}>
-      <Trash16 />
-    </Button>
+    {hasDeleteButton && (
+      <Button data-testid="id-delete-duplicate-group" type={ButtonType.Icon} disabled={IS_DISABLED_FOR_ALPHA}>
```
Comment (askhat-abishev) [inline src/components/DuplicateGroup/DuplicateGroup.tsx:21]: do we still need IS_DISABLED_FOR_ALPHA flag?

File: src/common/helpers/repeatableFields.helper.ts (hunk 1 above)
Comment (SKarolFolio) [reply src/common/helpers/repeatableFields.helper.ts:54]: May be, but we still have "repeatable" value as "true" for "Search place of publication (to/from 008)"

File: src/components/DuplicateGroup/DuplicateGroup.tsx (hunk 2 above)
Comment (SKarolFolio) [reply src/components/DuplicateGroup/DuplicateGroup.tsx:21]: I don't think so.
And this is out of the scope of the current task. I think, this button will be enabled in the scope of implementation of the required functionality.

**Review by askhat-abishev — APPROVED** [review]

**Pull request merged**